nodejs/week1/mealSharingApp/src/backend/routes/meal.js

```diff
@@ -0,0 +1,14 @@
+const dataHelper = require('../data/dataHelper')
+
+module.exports = (req, res) => {
+
+    const meals = dataHelper.getMeals();
+
+    const randomMealIndex =  Math.floor(Math.random()* meals.length);
+    
+    const meal = meals[randomMealIndex];
+    
+    const result = dataHelper.getMealsAugmentedWithReviews([meal])
+
+    res.send(result);
```

Comment:
This route returns a JSON array with a single meal object, so I would probably change this to return the object only.

Reply:
done it.